src/routes/dev/powerOfTwo.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const { isPowerOfTwo } = require( '../../utils/index' );
```

Comment:
Tip - `index` isn't required here because it'll automatically try to import from index.js when the filename is omitted.

ex:
```
const { isPowerOfTwo } = require( '../../utils' );